zio-http/src/main/scala/zio/http/Http.scala

```diff
@@ -36,14 +36,20 @@ sealed trait Http[-R, +Err, -In, +Out] { self =>
    */
   final def >>>[R1 <: R, Err1 >: Err, In1 >: Out, Out1](
     handler: Handler[R1, Err1, In1, Out1],
+    errorOutputMapper: Out => Out1,
```

Comment:
This operator became weird by this parameter, and in fact - at least in zio-http's codebase - every time it is used, `Out1` is `Unit`. So maybe we should replace it with a stricter, single parameter version.

Reply:
Yes, I agree. This is a bit too weird!